internal/nodes/avi_model_l4_translator.go

```diff
@@ -441,10 +441,17 @@ func getAutoFQDNForService(svcNamespace, svcName string) string {
 	if lib.GetL4FqdnFormat() == lib.AutoFQDNDefault {
 		// Generate the FQDN based on the logic: <svc_name>.<namespace>.<sub-domain>
 		fqdn = svcName + "." + svcNamespace + "." + subdomain
+
 	} else if lib.GetL4FqdnFormat() == lib.AutoFQDNFlat {
 		// Generate the FQDN based on the logic: <svc_name>-<namespace>.<sub-domain>
 		fqdn = svcName + "-" + svcNamespace + "." + subdomain
 	}
+	if !lib.CheckRFC1035(fqdn) {
+		if lib.CheckRFC1035("svc" + fqdn) {
+			fqdn = "svc" + fqdn
```

Comment:
Shall we have a hyphen between string "svc" and fqdn so that it will become more readable?